src/map/ai/states/synth_state.cpp

```diff
@@ -0,0 +1,112 @@
+/*
+===========================================================================
+
+  Copyright (c) 2025 LandSandBoat Dev Teams
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see http://www.gnu.org/licenses/
+
+===========================================================================
+*/
+
+#include "synth_state.h"
+
+#include "entities/battleentity.h"
+
+#include "ai/ai_container.h"
+#include "packets/action.h"
+#include "packets/lock_on.h"
+#include "utils/battleutils.h"
+#include "utils/synthutils.h"
+
+CSynthState::CSynthState(CCharEntity* PChar, SKILLTYPE skill)
+: CState(PChar, PChar->targid)
+, m_PEntity(PChar)
+{
+    if (m_errorMsg)
```

Comment:
Is it possible for this state to be created with errorMsg populated?

Reply:
Not sure, actually

Reply:
Removed the error handling, I don't think it's possible to hit anyway.